hub_content/policy_packs/gcp/kms/check_crypto_key_rotated_regularly/policies.tf

```diff
@@ -0,0 +1,39 @@
+# GCP > KMS > Crypto Key > Approved
+resource "turbot_policy_setting" "gcp_kms_crypto_key_approved" {
+  resource = turbot_smart_folder.pack.id
+  type     = "tmod:@turbot/gcp-kms#/policy/types/cryptoKeyApproved"
+  value    = "Check: Approved"
+}
+
+# GCP > KMS > Crypto Key > Approved > Custom
+resource "turbot_policy_setting" "gcp_kms_crypto_key_approved_custom" {
+  resource       = turbot_smart_folder.pack.id
+  type           = "tmod:@turbot/gcp-kms#/policy/types/cryptoKeyApprovedCustom"
+  template_input = <<-EOT
+    {
+      cryptoKey {
+        rotationPeriod: get(path: "rotationPeriod")
+        keyState: get(path: "primary.state")
+      }
+    }
+    EOT
+  template       = <<-EOT
+    {%- if $.cryptoKey.keyState == "DISABLED" or $.cryptoKey.keyState == "DESTROY_SCHEDULED" or $.cryptoKey.keyState == "DESTROYED" %}
+      title: Rotation
+      result: Skip
+      message: Crypto key is {{ $.cryptoKey.keyState }}
+    {%- else %}
+      {%- set rotationPeriodWithoutSeconds = $.cryptoKey.rotationPeriod.slice(1, -1) -%}
+      {%- set rotationPeriodInDays = rotationPeriodWithoutSeconds / 86400 -%}
+      {%- if rotationPeriodInDays > 1 %}
```

Comment:
@Subhajit97 I think we can set the time period of rotation to 90 days as in the GCP > CIS v2.0 it states that `1.10 Ensure KMS Encryption Keys Are Rotated Within a Period of 90 Days`.

Reply:
Right @niharika2k00, I have pushed the commit with the updated rotation period value, thanks!